examples/sdl_opengl/main.odin

```diff
@@ -68,14 +68,14 @@ main :: proc() {
                 imsdl.process_event(e, &imgui_state.sdl_state);
                 #partial switch e.type {
                     case .Quit:
-                        log.info("Got SDL_QUIT event!");
-                        running = false;
+//                        log.info("Got SDL_QUIT event!");
+//                        running = false;
 
                     case .Key_Down:
-                        if is_key_down(e, .Escape) {
-                            qe := sdl.Event{};
-                            qe.type = .Quit;
-                            sdl.push_event(&qe);
+//                        if is_key_down(e, .Escape) {
+//                            qe := sdl.Event{};
+//                            qe.type = .Quit;
+//                            sdl.push_event(&qe);
                         }
```

Comment:
This line needs to be commented out too, otherwise it introduces syntax errors.

(while I also not sure that this "commenting out" actually should be included in the PR, perhaps these are some forgotten "debug" changes.